### PR TITLE
enh(AC,ACC):  translation and ux enhancements

### PR DESCRIPTION
**Pull request by Med1233**
## Description

fixes and enhancements:
 - Delete icon on connecteurs List is red, should be blue like in other menus, and red on hover with a tooltip.
 - Increase nabber menu width from 192px to 208px
 - fix transaltion 

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] master

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by Thebarda — APPROVED** [review]

**Review by quborg — APPROVED** [review]

**Review by dmyios — DISMISSED** [review]

File: centreon/www/install/php/Update-24.10.3.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+// ACC
+$fixNamingOfAccTopology = function (CentreonDB $pearDB) use (&$errorMessage): void {
+    $errorMessage = 'Unable to retrieve data from informations table';
+    $statement = $pearDB->executeQuery(
+        <<<'SQL'
+            SELECT `value` FROM `informations` WHERE `key` = 'isCentral'
+            SQL
+    );
+    $isCentral = $statement->fetch(PDO::FETCH_COLUMN);
+
```
Comment (a-launois) [inline centreon/www/install/php/Update-24.10.3.php:40]:
```suggestion
```

File: centreon/www/install/php/Update-24.10.3.php [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    $errorMessage = 'Unable to update table topology';
+    $constraintStatement = $pearDB->prepareQuery(
+        <<<'SQL'
+            UPDATE `topology`
+            SET `topology_name` = 'Additional connector configurations'
+            WHERE `topology_url` = '/configuration/additional-connector-configurations'
+            SQL
+    );
+    $pearDB->executePreparedQuery($constraintStatement, [':show' => $isCentral === 'yes' ? '1' : '0']);
```
Comment (a-launois) [inline centreon/www/install/php/Update-24.10.3.php]:
```suggestion
    $pearDB->executePreparedQuery($constraintStatement);
```

File: centreon/www/install/php/Update-24.10.3.php [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+};
+
+
+try {
+    $addDisableServiceCheckColumn($pearDB);
```
Comment (a-launois) [inline centreon/www/install/php/Update-24.10.3.php:53]:
```suggestion
```

**Review by a-launois — CHANGES_REQUESTED** [review]

File: centreon/www/install/php/Update-24.10.3.php [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    $errorMessage = 'Unable to retrieve data from informations table';
+    $statement = $pearDB->executeQuery(
+        <<<'SQL'
+            SELECT `value` FROM `informations` WHERE `key` = 'isCentral'
+            SQL
+    );
+    $isCentral = $statement->fetch(PDO::FETCH_COLUMN);
+
+    $errorMessage = 'Unable to update table topology';
+    $constraintStatement = $pearDB->prepareQuery(
```
Comment (a-launois) [inline centreon/www/install/php/Update-24.10.3.php:42]:
```suggestion
    $constraintStatement = $pearDB->executeQuery(
```

**Review by a-launois — CHANGES_REQUESTED** [review]

File: centreon/cypress/visual-testing-snapshots/www/front_src/src/AgentConfiguration/AgentConfigurations.cypress.spec.tsx/Agent configurations modal -- adds a new host configuration when the corresponding button is clicked.snap.png
Comment (Thebarda) [inline centreon/cypress/visual-testing-snapshots/www/front_src/src/AgentConfiguration/AgentConfigurations.cypress.spec.tsx/Agent configurations modal -- adds a new host configuration when the corresponding button is clicked.snap.png:1]: The scroll in host configurations disappeared so you cannot scroll within host configurations

**Comment by Thebarda** [comment]
And changes are made on AC page, not on ACC

File: centreon/cypress/visual-testing-snapshots/www/front_src/src/AgentConfiguration/AgentConfigurations.cypress.spec.tsx/Agent configurations modal -- adds a new host configuration when the corresponding button is clicked.snap.png
Comment (Med1233) [reply centreon/cypress/visual-testing-snapshots/www/front_src/src/AgentConfiguration/AgentConfigurations.cypress.spec.tsx/Agent configurations modal -- adds a new host configuration when the corresponding button is clicked.snap.png:1]: oops

**Comment by Med1233** [comment]
> And changes are made on AC page, not on ACC

In fact, on both pages

File: centreon/www/install/php/Update-24.10.3.php [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+// ACC
+$fixNamingOfAccTopology = function (CentreonDB $pearDB) use (&$errorMessage): void {
+    $errorMessage = 'Unable to update table topology';
+    $constraintStatement = $pearDB->executeQuery(
+        <<<'SQL'
+            UPDATE `topology`
+            SET `topology_name` = 'Additional connector configurations'
+            WHERE `topology_url` = '/configuration/additional-connector-configurations'
+            SQL
+    );
```
Comment (Med1233) [inline centreon/www/install/php/Update-24.10.3.php:40]:
```suggestion
    );
    $isCentral = $statement->fetch(PDO::FETCH_COLUMN);
```

Comment (Med1233) [inline centreon/www/install/php/Update-24.10.3.php:40]:
```suggestion
    );
    $pearDB->executePreparedQuery($constraintStatement);
```

File: centreon/cypress/visual-testing-snapshots/www/front_src/src/AgentConfiguration/AgentConfigurations.cypress.spec.tsx/Agent configurations modal -- adds a new host configuration when the corresponding button is clicked.snap.png
Comment (Med1233) [reply centreon/cypress/visual-testing-snapshots/www/front_src/src/AgentConfiguration/AgentConfigurations.cypress.spec.tsx/Agent configurations modal -- adds a new host configuration when the corresponding button is clicked.snap.png:1]: I just checked that the cursor does exist, but it's not visible

Comment (Med1233) [reply centreon/cypress/visual-testing-snapshots/www/front_src/src/AgentConfiguration/AgentConfigurations.cypress.spec.tsx/Agent configurations modal -- adds a new host configuration when the corresponding button is clicked.snap.png:1]: I suppose it's related to the OS

**Review by dmyios — APPROVED** [review]

File: centreon/www/install/php/Update-24.10.3.php [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+    $pearDB->executePreparedQuery($constraintStatement);
```
Comment (a-launois) [inline centreon/www/install/php/Update-24.10.3.php:41]:
```suggestion
```

**Pull request merged**